ui/components/ui/popover/popover.component.js

```diff
@@ -135,8 +145,9 @@ const Popover = ({
             backgroundColor={BackgroundColor.backgroundDefault}
             color={Color.iconDefault}
             onClick={onScrollDownButtonClick}
-            className="whats-new-popup__scroll-button"
-            data-testid="whats-new-popup-scroll-button"
+            className="popover-scroll-button"
+            style={{ bottom: footer ? '140px' : '12px' }}
+            data-testid="popover-scroll-button"
```

Comment:
Question: Since this is a Button with an icon does it make sense to use ButtonIcon component here or at least can we convert this Box to button component?

Reply:
I spent a good amount of time trying to make this happen but I'll have to revisit it another time. Thanks for the suggestion!